src/models/Settings.php

```diff
@@ -42,6 +42,9 @@ class Settings extends Model
     /* @var int */
     public $batch_size = 1000;
 
+    /** @var bool */
+    public $useOriginalRecordIfSplitValueIsArrayOfOne = true;
```

Comment:
May also be worth adding it to the config file:

https://github.com/vigetlabs/craft-scout/blob/master/config/scout.php

Reply:
good all, & a chance to add a better explanation